api/v1/model/src/commonMain/kotlin/OrganizationVulnerability.kt

```diff
@@ -45,5 +45,8 @@ data class OrganizationVulnerability(
      * Count of repositories that the [Vulnerability] is found in. Relative to the restrictions of the query that
      * provides the information.
      */
-    val repositoriesCount: Long
+    val repositoriesCount: Long,
+
+    /** The purl of the package that the [Vulnerability] is found in. */
+    val purl: String
```

Comment:
@mnonnenmacher and @lamppu, is there a need / convention to add new properties for these API classes only at the end of the class? Otherwise putting `purl` next to `identifier` would probably be more readable.